types/plotly.js/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for plotly.js 1.54
+// Type definitions for plotly.js 2.4
```

Comment:
this requires removals from api to be implemented here (short list really):
https://github.com/plotly/plotly.js/releases/tag/v2.0.0
(removed section)